warmup.krun

```diff
@@ -6,7 +6,7 @@ from krun.vm_defs import (PythonVMDef, LuaVMDef, JavaVMDef, GraalVMDef,
 from krun import EntryPoint
 
 # Who to mail
-MAIL_TO = ["vext01@gmail.com", "mount.sarah@gmail.com"]
+MAIL_TO = []
 
 # Maximum number of error emails to send per-run
 #MAX_MAILS = 2
```

Comment:
Why is this line commented? [I know it isn't changed, but I only just spotted it!]